recipes/luarocks/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+
+
+# TODO
+# - Would be good if luarocks would offer the ability to use only a user rockspec, instead of system.
+#   (Using system-only rockspec with --force-config flag installs packages with restrictive privileges)
+./configure --prefix=$PREFIX \
+    --sysconfdir=$PREFIX/share/lua/ \
+    --with-lua-include=$PREFIX/include \
+    --rocks-tree=$PREFIX
+
+make bootstrap
```

Comment:
What do you think the functional difference is between a usual install and installing as a rock?